experimenter/experimenter/experiments/models.py

```diff
@@ -1347,6 +1344,170 @@ def get_branch_data(self, analysis_basis, selected_segment):
 
         return branch_data
 
+    def get_metric_area_data(
+        self, metrics, analysis_basis, segment, reference_branch, window="overall"
+    ):
+        metric_data = {}
+        metric_area_data = {"metrics": metrics, "data": metric_data}
+
+        window_results = self.get_window_results(analysis_basis, segment, window)
+
+        for branch in self.get_sorted_branches():
+            branch_results = (
+                window_results.get(branch.slug, {}).get("branch_data", {})
+                if isinstance(window_results, dict)
+                else {}
+            )
+            branch_metrics = {}
+
+            for metric in metrics:
+                slug = metric.get("slug")
+                group = metric.get("group")
+
+                metric_src = branch_results.get(group, {}).get(slug, {})
+
+                absolute_data_list = metric_src.get("absolute", {}).get("all", [])
+                relative_data_list = (
+                    metric_src.get("relative_uplift", {})
+                    .get(reference_branch, {})
+                    .get("all", [])
+                )
+
+                def formatted_analysis_point_comparator(point):
+                    wi = point.get("window_index")
+                    return int(wi) if wi is not None else 0
+
+                absolute_data_list.sort(key=formatted_analysis_point_comparator)
+                relative_data_list.sort(key=formatted_analysis_point_comparator)
+
+                significance_map = (
+                    metric_src.get("significance", {})
+                    .get(reference_branch, {})
+                    .get(window, {})
+                )
+
+                abs_entries = []
+                for i, data_point in enumerate(absolute_data_list):
+                    lower = round(data_point.get("lower"), 2)
+                    upper = round(data_point.get("upper"), 2)
+
+                    if metric.get("display_type") == "percentage":
+                        lower = f"{lower * 100}%"
+                        upper = f"{upper * 100}%"
+
+                    significance = significance_map.get(str(i + 1), "neutral")
+                    abs_entries.append(
+                        {"lower": lower, "upper": upper, "significance": significance}
+                    )
+
+                rel_entries = []
+                for i, data_point in enumerate(relative_data_list):
+                    lower = data_point.get("lower")
+                    upper = data_point.get("upper")
+                    avg_rel_change = abs(round((lower + upper) / 2 * 100))
```

Comment:
I think we can just get the `point` value from the results here, no? And then multiply by 100 if you want a percentage.